_posts/0000-01-02-zeezar4.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
```

Comment:
https://www.google.com/url?sa=i&url=https%3A%2F%2Fwww.pinterest.com%2Ftrynitythompson%2Fget-a-load-of-this-guy%2F&psig=AOvVaw3gTZJ6EI8mpxhVul1XHhr3&ust=1589387445920000&source=images&cd=vfe&ved=0CAIQjRxqFwoTCKj_4-LfrukCFQAAAAAdAAAAABAD